src/m365/aad/commands/o365group/o365group-add.ts

```diff
@@ -179,6 +138,95 @@ class AadO365GroupAddCommand extends GraphCommand {
       }, (rawRes: any): void => this.handleRejectedODataJsonPromise(rawRes, logger, cb));
   }
 
+
+  private validateOwners(logger: Logger, args: CommandArgs): Promise<any> {
```

Comment:
As the method returns an array of user IDs, let's rename it to make it clear, to something like `getUserIds`